interface/wx/aui/framemanager.h

```diff
@@ -765,11 +765,25 @@ class wxAuiPaneInfo
     //@{
     /**
         FloatingSize() sets the size of the floating pane.
+        FloatingClientSize() has precedence over this.
```

Comment:
I think we need a blank line here for the docs to appear correctly, or did Doxygen relax its rules about it? In any case, it doesn't hurt the readability and we could also improve the description a bit.

```suggestion

        FloatingClientSize() has precedence over this, i.e. this size is ignored
        if the floating client size is specified.
```

(also below)

Reply:
I have never used Doxygen, and have no reason to think its syntax has changed, so I will apply your suggestions here and elsewhere.

Reply:
Done.  See 8c9c2d608514df13b13a486553d100f492762d34.